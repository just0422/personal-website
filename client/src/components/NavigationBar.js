import React, { Component } from 'react';
import { Navbar, NavItem, Nav  } from 'react-bootstrap';

export default class NavigationBar extends Component {
	render () {
		return (
			<div>
				<Navbar inverse>
					<Navbar.Header>
						<Navbar.Brand>
							<a href="#brand">React-Bootstrap</a>
						</Navbar.Brand>
						<Navbar.Toggle />
					</Navbar.Header>
					<Navbar.Collapse>
						<Nav pullRight>
							<NavItem eventKey={1} href="/">
								Resume
							</NavItem>
							<NavItem eventKey={2} href="/projects">
								Projects
							</NavItem>
							<NavItem eventKey={3} href="/about">
								About
							</NavItem>
							<NavItem eventKey={4} href="/contact">
								Contact
							</NavItem>
						</Nav>
					</Navbar.Collapse>
				</Navbar>
			</div>
		);
	}
}
