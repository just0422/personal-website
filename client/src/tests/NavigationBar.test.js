import React from 'React';
import {shallow} from 'enzyme';

import NavigationBar from 'NavigationBar';

describe("Navigation bar tests", () => {
	it('should render correctly', () => {
		const component = shallow(<NavigationBar />);
		expect(component).toMatchSnapshot();
	});
});
