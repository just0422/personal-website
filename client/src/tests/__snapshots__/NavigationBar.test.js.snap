// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation bar tests should render correctly 1`] = `ShallowWrapper {}`;
